packages/node/src/events/install/controller.ts

```diff
@@ -18,3 +18,5 @@ export async function installEventController(
     nodeMsg.data.params
   );
 }
+
+export default installEventController;
```

Comment:
Why not just leave `default` in the function signature?

Saves having to export down here.

Reply:
Now it is exporting both as a symbol _and_ default, so importers have a choice

Reply:
I mean, this is what `defaults` are for really - to indicate that a file only has one thing to import from, so the `import { x } from './x'` seems like an antipattern to `default` usage.

Reply:
Hmmm, ok I'll remove this